README.md

```diff
@@ -47,9 +47,11 @@ Station Core is configured using environment variables (see
 
 The following configuration options are shared by all Station commands:
 
-- `$XDG_STATE_HOME` _(string; optional)_: Station stores logs and module state
-  in the directory `$XDG_STATE_HOME/filecoin-station`. State home defaults to
-  `~/.local/state`.
+- `$STATION_ROOT`_(string; optional)_: Station stores logs and module state in
```

Comment:
Should we use `ROOT_DIR` for consistency with Station Modules and Zinnia, instead of `STATION_ROOT`?

When running Station Core standalone, we want to tell it where to keep the files (what is the root dir); there isn't really a concept of a station root, is it?